kornia/color/hls.py

```diff
@@ -0,0 +1,146 @@
+import torch
+import torch.nn as nn
+import cv2
+import kornia
+
+
+class HlsToRgb(nn.Module):
+    r"""Convert image from HLS to Rgb
+    The image data is assumed to be in the range of (0, 1).
+
+    args:
+        image (torch.Tensor): HLS image to be converted to RGB.
+
+    returns:
+        torch.tensor: RGB version of the image.
+
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 3, H, W)`
+
+    Examples::
+
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> rgb = kornia.color.HlsToRgb()
+        >>> output = rgb(input)  # 2x3x4x5
+
+    """
+
+    def __init__(self) -> None:
+        super(HlsToRgb, self).__init__()
+
+    def forward(self, image: torch.Tensor) -> torch.Tensor:  # type: ignore
+        return hls_to_rgb(image)
+
+
+def hls_to_rgb(image):
+    r"""Convert an HSV image to RGB
+    The image data is assumed to be in the range of (0, 1).
+
+    Args:
+        input (torch.Tensor): RGB Image to be converted to HSV.
+
+
+    Returns:
+        torch.Tensor: HSV version of the image.
+    """
+
+    if not torch.is_tensor(image):
+        raise TypeError("Input type is not a torch.Tensor. Got {}".format(
+            type(image)))
+
+    if len(image.shape) < 3 or image.shape[-3] != 3:
+        raise ValueError("Input size must have a shape of (*, 3, H, W). Got {}"
+                         .format(image.shape))
+
```

Comment:
Wondering whether we should check if input is in floating point precision. @ducha-aiki